test/verify/check-packagekit

```diff
@@ -39,7 +39,7 @@ done
 
 OSesWithoutTracer = ["debian-stable", "debian-testing", "ubuntu-2204", "ubuntu-stable", "fedora-coreos"]
 OSesWithoutKpatch = ["debian-stable", "debian-testing", "ubuntu-2204", "ubuntu-stable",
-                     "fedora-coreos", "fedora-35", "fedora-36", "fedora-testing", "centos-8-stream", "arch"]
+                     "fedora-coreos", "fedora-35", "fedora-36", "fedora-37", "fedora-testing", "centos-8-stream", "arch"]
```

Comment:
do we still keep 35 image and testing?

Reply:
Dropping this is the next step, for now I want to enable f37.